src/portal/lib/nodes.ts

```diff
@@ -422,9 +429,15 @@ export async function getDNodes(
   }[] = [];
   for (const node of nodes) {
     const price = countPrice(pricing, node);
-    const [discount, discountLevel] = await calDiscount(api, address, pricing, price);
-    const ips = await getIpsForFarm(node.farmId);
+    const [discount, discountLevel] = await calDiscount(TFTbalance, pricing, price);
+    const ips = "";
```

Comment:
You changed this to be an empty string while you should request to get the number of public ips.

```ts
const ips = await getIpsForFarm(node.farmId);
dNodes.push({
    pubIps: ips,
})
```

Reply:
the idea is, instead of fetching all nodes data, we set the data that will be fetched as empty, and on click on a node, the process of loading <fetching> farm data and replace the empty data got started

Reply:
okay, but why do you declare a constant called ips and its value is an empty string then you pass it at the end of the object?